src/Sentry/Laravel/Tracing/EventHandler.php

```diff
@@ -83,6 +90,7 @@ public function __construct(array $config)
     {
         $this->traceSqlQueries = ($config['sql_queries'] ?? true) === true;
         $this->traceSqlQueryOrigins = ($config['sql_origin'] ?? true) === true;
+        $this->recordSqlBindings = ($config['sql_bindings'] ?? true) === true;
```

Comment:
Can you move this below `$this->traceSqlQueries`?

Reply:
I've gone ahead and made those changes 👍